framework/src/modules/dpos/db.ts

```diff
@@ -0,0 +1,74 @@
+/*
```

Comment:
maybe file name should be `data_access`? i think in other cases we use this file name